### ISSUE TITLE
FOTT: "TypeError" and no files load to left pane for OCR

### ISSUE THREAD

**Issue by glarose13** [microsoft/OCR-Form-Tools#290]
**Describe the bug**
- using https://fott.azurewebsites.net/
- javascript enabled for Microsoft Edge, FireFox, Google Chrome
- On "Save Project" error message: "TypeError cannot read property 'split' of undefined" appears in Tag Editor window
- "Successfully saved" project settings notification appears in the window
- On clicking "Ok" in error box,  FOTT appears to hang, with waiting cursor running continuous
- *.fott file written to blob container successfully
- files do not load to left pane under "Run OCR on all files"

**To Reproduce**
1. Go to "New Project
2. Fill in Project Settings fields
3. Leave "Folder path" field blank (I have no subfolder in blob container)
4. Save Project

**Expected behavior**
FOTT to load in files to OCR in left pane

**Note**
- files in blob container are jpg
- if value added to folder path field in Project Settings,  "TypeError" does not occur
- no files load in left pain
- if tags added in tag editor window, folder created from value in folder path field and fields.json written to that folder
- if jpg moved from container to new subfolder "TypeError" appears

**Screenshots Microsoft Edge**
![FOTT2](https://user-images.githubusercontent.com/53874486/83336312-70311580-a280-11ea-944f-19cba2431bfa.JPG)
![FOTT1](https://user-images.githubusercontent.com/53874486/83336323-863ed600-a280-11ea-9183-20c270a0a605.JPG)

**Screenshot FireFox**
![FOTT TypeError FireFox](https://user-images.githubusercontent.com/53874486/83336681-53e2a800-a283-11ea-8d60-5b0faaf40da5.JPG)

**Screenshot Chrome**
![FOTT TypeError Chrome](https://user-images.githubusercontent.com/53874486/83336799-67dad980-a284-11ea-8e10-1a52a6f86d1d.JPG)

**Screenshot error message from local install**
![FOTT_localapp](https://user-images.githubusercontent.com/53874486/83338782-8c8b7d00-a295-11ea-9fa2-326a61dc9c51.JPG)

**Desktop (please complete the following information):**
 - Windows 10
 - Microsoft Edge v 83.0.478.37 64-bit
- also tested with Firefox v 76.0.1 64-bit, Chrome v83.0.4103.61 64-bit
- also ran it locally from git clone as per instructions

**Additional context**
Azure Resource region for blob storage: Canada Central

**Comment by glarose13** [comment]
I looked at the code associated with error in assetService.ts, which has this comment:
 // If extension of a file was spoofed, we fetch only first 4 bytes of the file and read MIME type
- suspected possible encoding issue with *.jpg files, so resaved them as *.jpg, *.png, *.tiff using 3D Paint app.
- attempting to access *.jpg files returns the above error
- access to *.png was successful, no error
- access to *.tiff was successful, no TypeError, but warning:
![HSFFORM1-TIFFError](https://user-images.githubusercontent.com/53874486/83339348-f6f2ec00-a29a-11ea-9fb9-f737095e585a.JPG)


- I have checked Microsoft Form Recognizer documentation for technical details on image requirements and all the file formats meet the specification
- the *.jpgs have previously processed without error via Read API v2 and v3 with handwriting recognition

My process is setup for jpeg.  Any guidance and advice on this is very much appreciated.
